instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/rum/RumConfigE2ETests.kt

```diff
@@ -668,6 +674,109 @@ class RumConfigE2ETests {
         sendAllRumEvents(forge, testMethodName)
     }
 
+    /**
+     * apiMethodSignature: com.datadog.android.core.configuration.Configuration$Builder#fun setVitalsUpdateFrequency(VitalsUpdateFrequency): Builder
+     */
+    @Test
+    fun rum_config_set_vitals_update_frequency_never() {
+        val testMethodName = "rum_config_set_vitals_update_frequency_never"
+        // we need to initialize the SDK on the Main Thread (Looper thread) otherwise the
+        // Choreographer Callback will not be registered
+        InstrumentationRegistry.getInstrumentation().runOnMainSync {
+            measureSdkInitialize {
+                initializeSdk(
+                    InstrumentationRegistry.getInstrumentation().targetContext,
+                    TrackingConsent.GRANTED,
+                    Configuration
+                        .Builder(
+                            logsEnabled = true,
+                            tracesEnabled = true,
+                            rumEnabled = true,
+                            crashReportsEnabled = true
+                        )
+                        .useViewTrackingStrategy(
+                            ActivityViewTrackingStrategy(
+                                true,
+                                componentPredicate = object : ComponentPredicate<Activity> {
+                                    override fun accept(component: Activity): Boolean {
+                                        return true
+                                    }
+
+                                    override fun getViewName(component: Activity): String {
+                                        return testMethodName
+                                    }
+                                }
+                            )
+                        )
+                        .setVitalsUpdateFrequency(VitalsUpdateFrequency.NEVER)
+                        .build()
+                )
+            }
+        }
+        val scenario = launch(ViewTrackingActivity::class.java)
+        // Give some time to vitals monitors to be updated
+        Thread.sleep(5000)
+        scenario.onActivity {
+            InstrumentationRegistry.getInstrumentation().callActivityOnPause(it)
```

Comment:
why do we need to call pause and then resume?

Reply:
Just because I want to make sure I get some choreographer updates.

Reply:
I was thinking that `Choreographer` always does `postFrameCallback`, even if nothing happens on the screen (so it is sufficient only to `launch` an activity), but maybe my knowledge is wrong.